tests/profiles/mysql/test_mysql_user_pass.py

```diff
@@ -0,0 +1,168 @@
+"""Tests for the mysql profile."""
+
+from unittest.mock import patch
+
+import pytest
+from airflow.models.connection import Connection
+
+from cosmos.profiles import get_automatic_profile_mapping
+from cosmos.profiles.mysql.user_pass import (
+    MysqlUserPasswordProfileMapping,
+)
+
+
+@pytest.fixture()
+def mock_mysql_conn():  # type: ignore
+    """
+    Sets the connection as an environment variable.
+    """
+    conn = Connection(
+        conn_id="my_mysql_connection",
+        conn_type="mysql",
+        host="my_host",
+        login="my_user",
+        password="my_password",
+        port=3306,
+        schema="my_database",
+    )
+
+    with patch("airflow.hooks.base.BaseHook.get_connection", return_value=conn):
```

Comment:
This will not work with Airflow 3.1. Please, could you update it to follow what we did in this commit?
https://github.com/astronomer/astronomer-cosmos/pull/1980/commits/0074a5ef495178853fd916ea9bdb05237754fd55